### ISSUE TITLE
Autodetect the configuration file, so that the script can be run from the command line

### ISSUE THREAD

**Issue by ephread** [ephread/Autodazzler#1]
At the moment, users must manually provide the configuration file though an interactive file dialog. An improvement would be to load the configuration file automatically if it's store in the same directory as the DSA script.

**Comment by ephread** [comment]
Could also take advantage of `DzApp.scriptArgs`, to provide arguments.

(macOS)
```bash
$ /Applications/DAZ\ 3D/DAZStudio4\ 64-bit/DAZStudio.app/Contents/MacOS/DAZStudio Test.duf -scriptArg "autodazzlerConfig='~/autodazzler.json'"
```

**Comment by ephread** [comment]
Implemented as of 0.2.0
